OutOfSchool/OutOfSchool.AuthorizationServer/Startup.cs

```diff
@@ -84,6 +84,7 @@ public static void AddApplicationServices(this WebApplicationBuilder builder)
             c.Cookie.Name = "OpenIdDict.Cookie";
             c.LoginPath = "/Auth/Login";
             c.LogoutPath = "/Auth/Logout";
+            c.ExpireTimeSpan = TimeSpan.FromDays(30);
```

Comment:
maybe make it configurable in `appsettings.json`, so we have the ability to change it without re-compilation.